src/core/api/index.js

```diff
@@ -285,16 +286,21 @@ export function logOutFromServer({ api }: {| api: ApiState |}) {
 }
 
 export type AutocompleteParams = {|
+  _fixFiltersForClientApp?: typeof fixFiltersForClientApp,
   api: ApiState,
   filters: {|
     query: string,
     addonType?: string,
   |},
 |};
 
-export function autocomplete({ api, filters }: AutocompleteParams) {
+export function autocomplete({
+  _fixFiltersForClientApp = fixFiltersForClientApp,
```

Comment:
I'm probably missing something obvious, but why are we doing it like this - can't `fixFiltersForClientApp` be used directly without having it as a parameter with an underscore?

Reply:
That is dependency injection, mainly for testing purposes.

We could use it directly but we wouldn't be able to easily create a mock for this function in a test case.

Reply:
for tests, got it.